ETL/Repository/QueryGenerator.cs

```diff
@@ -3,49 +3,47 @@
 
 namespace ETL.Repository
 {
-    public static class QueryGenerator
```

Comment:
Why modify this file?

Reply:
I think it just auto formatted when I looked at it